drum.go

```diff
@@ -0,0 +1,44 @@
+// Package drum is supposed to implement the decoding of .splice drum machine files.
+// See golang-challenge.com/go-challenge1/ for more information
+package drum
+
+import "fmt"
+
+const measureCount = 16
```

Comment:
It would be good to see some of the other magic numbers in this codebase get this treatment.